examples/sample-app/application-template-stibuild.json

```diff
@@ -356,7 +356,7 @@
               "failurePolicy": "Ignore",
               "execNewPod": {
                 "command": [
-                  "/bin/false"
+                  "/bin/true"
```

Comment:
no idea. guessing someone was trying to test the behavior of failed post deploy hooks.